content/docs/edit-site/edit-site.md

```diff
@@ -0,0 +1,7 @@
+# Editing this site
+
+If you want to edit or add to any of the pages here, please click on the "Edit this page" link at the bottom of the page you want to edit. This will take you to a GitHub repository, which stores the source files for that particular page. From there you can click on the tiny pencil (on hover: Edit this file), which will allow you to make any edits you want. Please remember that any tables or fancy formatting will have to be written in markdown / html to be visible. If you need any help with this please [get in touch](mailto:sonia.mitchell@glasgow.ac.uk). Once you're finished, just click the link at the bottom of the page to create a new branch and start a pull request.
```

Comment:
I've added a hopefully adequate bit of text. I'll try and figure out how to give all organisation member write access so @soniamitchell Sonia isn't too distracted by requests.

Reply:
Do we want everyone to have write access? If so, @richardreeve likely has a group set up for that..

Reply:
I think they'd need write access to make a branch to get to the PR stage (unless they'd forked) then we'd want an approving review and we could potentially restrict who was authorised to approve changes. (I do get a bit lost with all the permissions on GitHub sometimes, they're quite complex but often don't let me do exactly what I want!)

Reply:
My only concern is breaking the site, so whatever you think is best (I'm confident you have more experience than me). Perhaps a dev branch that everyone can write to? Though part of me thinks most people might be more comfortable emailing Rmarkdown scripts / Word documents, I agree it would be easier if people could just fend for themselves.

Reply:
I wouldn't be too confident of that! I suggest that the model is people make a branch, make changes, PR - this is reviewed by someone who can build the site locally (to make sure there are no breaking changes) and by someone who can offer a second opinion on the content. 

I think doing things by email would be great for people not comfortable with the above - we could make the branch and PR on their behalf.

Ultimately, if we get a lot of traffic, we might want to set up a CI step which does a test build of the site during the PR review process.

Reply:
Sounds like a good plan! Though from my experience last year, I think you're being a bit optimistic about the traffic :p

Reply:
Alright, I've set most of that up, giving me, yourself, and Ryan write access to master. Reviews set up for everyone else. Just need @richardreeve to give the "everyone" team write access generally.